galley/pkg/crd/validation/webhook.go

```diff
@@ -319,14 +334,17 @@ func (wh *Webhook) Run(stopCh <-chan struct{}) {
 		return
 	}
 
-	// Try to create the initial webhook configuration (if it doesn't
-	// already exist). Setup a persistent monitor to reconcile the
-	// configuration if the observed configuration doesn't match
-	// the desired configuration.
-	if err := wh.rebuildWebhookConfig(); err == nil {
-		wh.createOrUpdateWebhookConfig()
+	webhookChangedCh := make(chan struct{}, 1000)
```

Comment:
`1000` is not necessary here. This variable won't be used anyway if the flag is false; and if it's true, it will be overridden in the block below.

I wish there could be a way to remove it completely from the `select` if the flag is false...